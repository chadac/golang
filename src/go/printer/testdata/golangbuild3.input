// other comment

// +build y

// yet another comment

package p

//golang:build x

func f()
