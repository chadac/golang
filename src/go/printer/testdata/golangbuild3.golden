// other comment

//golang:build x
// +build x

// yet another comment

package p

func f()
